packages/client/src/views/SysAdmin/Vsexports/VSExport.tsx

```diff
@@ -46,6 +50,29 @@ type VSExportProps = {
   items: VsExport[]
 }
 
+async function getPreSignedURL(fileName: string) {
+  if (fileName.startsWith(`/${window.config.MINIO_BUCKET}/`)) {
+    fileName = fileName.replace(`/${window.config.MINIO_BUCKET}/`, EMPTY_STRING)
+  }
+  const url = new URL(
+    `document/${fileName}`,
+    window.config.API_GATEWAY_URL
+  ).toString()
+  console.log(url)
+  try {
+    const res = await fetch(url, {
+      method: 'GET',
+      headers: {
+        Authorization: `Bearer ${getToken()}`
+      }
+    })
+    const resJSON = (await res.json()) as { presignedURL: string }
+    return resJSON.presignedURL
+  } catch (error) {
+    throw new Error('download_failed')
+  }
```

Comment:
Probably just better to throw the original error upwards, so you can remove the try-catch altogether